datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -1127,3 +1142,8 @@ def degeneralise_tag(self, tag):
             return '{}:{}'.format(new_name, value)
         else:
             return tag
+
+    def get_debug_metric_tags(self):
+        tags = ['check_name:{}'.format(self.name), 'check_version:{}'.format(self.check_version)]
+        tags.extend(self.instance.get('tags', []))
```

Comment:
so will this also have the effect of adding these instance tags to the memory profiling metrics?

Reply:
yes, which seems fine imo. wdyt?

Reply:
I think it's fine! Just wanted to note it